website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Release Notes - Podman Desktop 1.2
+description: Podman Desktop 1.2 has been released!
+slug: podman-desktop-release-1.2
+authors: [cdrage]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.2 Release! 🎉
+
+We're excited to announce the release of Podman Desktop version 1.2.0! This release includes several new features, bug fixes, and improvements to enhance your container management experience. Here's a summary of the key changes in this release:
+
+- **Protocol handler support**: Added support for protocol handlers such as `open podman-desktop:extension/myextension`
+- **Kubernetes context on the status bar**: Kube context can now appear on the status bar
+- **Added start/stop/delete buttons for Compose**: You can now make changes to an entire Compose group of containers
+- **Ability to rename an image**: Rename an image with a click of a button
```

Comment:
"Ability to name an image" > "Rename images"? Just more concise.

Reply:
Also the Compose story should be the top one